taworks/taform/templates/taform/application.html

```diff
@@ -211,7 +211,13 @@ <h3>Confirmation</h3>
 <p><b>To submit your application you must agree to the above terms.</b></p>
 
 <p id="p01">Please note you will not be able to make changes to your application once you submit.<p>
-<input type="submit" id ="btn" value="Submit" disabled>
+<input type="hidden" name="btn" value="Submit" />
+<input type="button" id ="btn" value="Submit" disabled>
```

Comment:
remove this if we aren't using it

I think we should also rename it from btn -> application-btn